index.html

```diff
@@ -37,14 +37,14 @@ <h1 class="short-presentation">I’m Sajib Arafat Siddiqui <br>Glad to see you!<
         </p>
 
         <footer>
-          <h3 class="hd3">LET'S CONNECT</h3>
+          <h2 class="hd3">LET'S CONNECT</h2>
           <div class="contact">
             <ul>
               <li class="icon_home"><a href="#"><img src="statics/images/twitter.png" alt="twitter link"></a></li>
-              <li class="icon_home"><a href="#"><img src="statics/images/Linkedin_icon.png" alt="linked link"></a></li>
-              <li class="icon_home"><a href="#"><img src="statics/images/M.png" alt="M"></a> </li>
+              <li class="icon_home"><a href="#"><img src="statics/images/Linkedin_icon.png" alt="linkedin link"></a></li>
+              <li class="icon_home"><a href="#"><img src="statics/images/M.png" alt="medium link"></a> </li>
               <li class="icon_home"><a href="#"><img src="statics/images/github.png" alt="github link"></a></li>
-              <li class="icon_home"><a href="#"><img src="statics/images/doits.png" alt="doit"></a></li>
+              <li class="icon_home"><a href="#"><img src="statics/images/doits.png" alt="doit link"></a></li>
```

Comment:
You have done great by adding `alt` attribute to all images on the website. Kindly note that the alternative text does not need to include the words "button", "link", or "image of". (Screen readers automatically provide that information.)

Reply:
resolved